pom.xml

```diff
@@ -36,7 +36,7 @@
 		<gwtPluginVersion>2.8.1</gwtPluginVersion>
 		<gwtp.version>1.6</gwtp.version>
 		<gin.version>2.1.2</gin.version>
-
+		<agnie.version>1.0-SNAPSHOT</agnie.version>
```

Comment:
Please remove version for reCaptcha, since it's now part of the code.

Reply:
Removed.